res/values-vi/strings.xml

```diff
@@ -1136,11 +1211,13 @@
     <string name="Secondary">Thứ hai</string>
     <string name="Both">Cả hai</string>
 
-    <string name="Enable__Multiskin">"Bật 2 Skin"</string>
-    <string name="MULTISKIN">2 TRANG PHỤC CÙNG LÚC</string>
+    <string name="multiskin_period_description" translatable="true">Specify multiskin change period in seconds (0-10). 0 will disable multiskin.</string>
+    <string name="Enable__Multiskin">"Bật\nMultiskin"</string>
+    <string name="multiskin_description" translatable="true">Multiskin lets you use two skins that alternate by themselves. You can choose how fast they will change. When you reach level 1.000, this feature is free. If you buy multiskin before level 1.000, you may be able to receive a refund after reaching the level.</string>
```

Comment:
This line is not based on `res/values` to establish a more polished untranslated text; unlike how it happens with English, the millenials are counted using a dot, and decimals are counted using a comma in Vietnamese, just like many other languages that use latin alphabet (Portuguese, Italian, French, Spanish, Indonesian, Kurdish etc.)